packages/orchestration/src/exos/chain-account-kit.js

```diff
@@ -125,49 +125,60 @@ export const prepareChainAccountKit = (zone, { watch, when }) =>
           return this.state.port;
         },
         executeTx() {
-          throw new Error('not yet implemented');
+          return asVow(() => {
+            throw new Error('not yet implemented');
+          });
```

Comment:
orthogonal to this PR, but I think our preference is `Error(...)`, not `new Error(...)`

```suggestion
          return asVow(() => {
            throw Error('not yet implemented');
          });
```

I'm struggling to confirm from [Coding Style](https://github.com/Agoric/agoric-sdk/wiki/Coding-Style) though.